src/styles/blocks/variables.scss

```
@@ -0,0 +1,3 @@
+$mainColor: #060b35;
+$greyColor: #616070;
+$blueColor: #00acdc;
```

it'd be more flexible
```suggestion
$neutralColor: #616070;
$accentColor: #00acdc;
```